bin/adhoc-scripts/createPileupObjects.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
```

Comment:
It is a good practice to add creator name in python code.

Reply:
I don't have a strong opinion about this, but I think it deserves a discussion between our team in our next weekly meeting.